pom.xml

```diff
@@ -38,13 +38,37 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <spotbugs.excludes.file />
         <surefire.forkCount>1C</surefire.forkCount>
+        <version.accumulo>2.1.1</version.accumulo>
+        <version.annotations>2.0.3</version.annotations>
+        <version.commons-codec>1.7</version.commons-codec>
+        <version.commons-collections4>4.4</version.commons-collections4>
+        <version.commons-lang3>3.12.0</version.commons-lang3>
         <version.docker.plugin>1.2.1</version.docker.plugin>
+        <version.guava>28.1-jre</version.guava>
+        <version.hadoop>3.3.5</version.hadoop>
+        <version.hazelcast>5.1.2</version.hazelcast>
         <version.jackson>2.13.2</version.jackson>
+        <version.jakarta-bind-api>2.3.3</version.jakarta-bind-api>
+        <version.javax.inject>1</version.javax.inject>
+        <version.jaxb>2.3.3</version.jaxb>
+        <version.jts-core>1.19.0</version.jts-core>
+        <version.junit>4.13.2</version.junit>
         <!-- NOTE: The Junit and log4j management here will NOT override the management provided via spring boot.  Make
         sure to update these versions as you update the spring-boot version so that we don't get out of date. -->
         <version.junit.bom>5.8.2</version.junit.bom>
         <version.log4j.bom>2.17.2</version.log4j.bom>
         <version.lombok>1.18.24</version.lombok>
+        <version.powermock>2.0.4</version.powermock>
+        <version.protobuf>2.5.0</version.protobuf>
+        <version.protostuff>1.6.2</version.protostuff>
+        <version.rabbitmq>2.1.0.RELEASE</version.rabbitmq>
+        <version.reload4j>1.2.22</version.reload4j>
+        <version.slf4j>1.7.29</version.slf4j>
+        <version.spring>5.3.21</version.spring>
+        <version.spring-boot>2.7.1</version.spring-boot>
+        <version.spring-cloud-context>3.1.3</version.spring-cloud-context>
+        <version.swagger-v3>2.2.0</version.swagger-v3>
+        <version.xml-apis>1.4.01</version.xml-apis>
     </properties>
     <dependencyManagement>
```

Comment:
So I expected to see additions to the dependency management section?

Reply:
Just pushed up a new one that does have the entries in the dependencyManagement section.